utils/config.go

```diff
@@ -394,6 +394,33 @@ const DefaultGrpcWcsConcPerNode = 16
 const DefaultWmsPolygonShardConcLimit = 5
 const DefaultWcsPolygonShardConcLimit = 10
 
+// Load dates for the ith layer
```

Comment:
Unit test case needed.

Reply:
Nar, not this time as this is not an entirely new function. `GenerateDatesMas()` already has test cases that cover it. The rest were cut and paste from `LoadConfigFile()`

Reply:
If you write a unit test case for every function as it is generated, you will end up with a testsuite with good coverage. As it is, `GetLayerDates` will have zero coverage.